codegen/tests/simple/app.go

```diff
@@ -0,0 +1,37 @@
+// Code generated by sysl DO NOT EDIT.
+package simple
+
+import (
+	"github.com/anz-bank/sysl-go/codegen/tests/deps"
+	"github.com/anz-bank/sysl-go/codegen/tests/downstream"
+	"github.com/anz-bank/sysl-go/config"
+	"github.com/anz-bank/sysl-go/core"
+	"github.com/anz-bank/sysl-go/handlerinitialiser"
+)
+
+// HandlerInit for Simple
+type HandlerInit struct {
+	enabledHandlers     []handlerinitialiser.HandlerInitialiser
```

Comment:
Can these renamed to:
```go
RestHandlers []handlerinitialiser.HandlerInitialiser // capitalised for export
GrpcHandlers []handlerinitialiser.GrpcHandlerInitialiser
```

We'll be pushing for some naming consistency in the codebase soon.

Reply:
ok